tools/ci-build/build-image.dockerfile

```diff
@@ -0,0 +1,124 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0.
+#
+ARG base_image=public.ecr.aws/amazonlinux/amazonlinux:2
+FROM ${base_image} AS bare_base_image
+
+#
+# Node Installation Stage
+#
+FROM bare_base_image AS install_node
+ARG node_version=v16.14.0
```

Comment:
For stuff like this, should we be using our ticket-creating bot to encouraged ourselves to bump this version with some regularity?